src/internal/m365/exchange/restore_test.go

```diff
@@ -369,3 +373,71 @@ func (suite *RestoreIntgSuite) TestRestoreExchangeObject() {
 		})
 	}
 }
+
+func (suite *RestoreIntgSuite) TestRestoreAndBackupEventWithRecurringAndAttachments() {
+	t := suite.T()
+
+	ctx, flush := tester.NewContext(t)
+	defer flush()
+
+	var (
+		userID  = tester.M365UserID(t)
+		subject = testdata.DefaultRestoreConfig("event").Location
+		handler = newEventRestoreHandler(suite.ac)
+	)
+
+	calendar, err := handler.ac.CreateContainer(ctx, userID, subject, "")
+	require.NoError(t, err, clues.ToCore(err))
+
+	calendarID := ptr.Val(calendar.GetId())
+
+	defer func() {
+		// Removes calendar containing events created during the test
+		err = suite.ac.Events().DeleteContainer(ctx, userID, calendarID)
+		assert.NoError(t, err, clues.ToCore(err))
+	}()
+
```

Comment:
we should be able to skip this. Plus we don't want to fail the test if everything but removing the calendar passes